aibolit/utils/filter.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Tuple, Any, Union, TypeVar, Type
+from javalang.tree import ClassDeclaration, InterfaceDeclaration, MethodDeclaration, \
+    MemberReference, FieldDeclaration, MethodInvocation, This, Node, LocalVariableDeclaration
+
+FldExh = Tuple[str, Tuple[str, str]]
```

Comment:
Data types you use in the program have to be declared just one time in a single place.